docs/developers/docker-node-key.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: Managing Wallets in Docker
+---
+
+# Docker and `cel-key`
+
+## Prerequisites
+
+First, familiarize yourself with the [guide](../nodes/docker-images.mdx) on how to run `celestia-node` with Docker.
+
+## Running your node
+
+Run the Docker image (in this example, we are using a Light Node):
+
+```bash
+docker run -e NODE_TYPE=light -e P2P_NETWORK=mocha -p 26659:26659 celestia-node celestia light start --core.ip https://rpc-mocha.pops.one --gateway --gateway.addr 127.0.0.1 --gateway.port 26659 --p2p.network mocha
```

Comment:
If you run this command and don't have the `celestia-node` image cached on your machine this command fails. Add the container registry: `ghcr.io/celestiaorg/celestia-node`

Reply:
Hm, I've had docker pull/download it even if I don't have it cached. Does this sound right?

Reply:
yes! thank you for catching this